catalog/spatial/registry/registry-federation-admin-service-impl/src/main/java/org/codice/ddf/registry/federationadmin/service/impl/RefreshRegistryEntries.java

```diff
@@ -166,27 +223,44 @@ private void writeRemoteUpdates(List<Metacard> remoteMetacardsToUpdate)
                 return null;
             });
         } catch (PrivilegedActionException e) {
-            String message = "Error writing remote updates.";
-            LOGGER.debug("{} Metacard IDs: {}", message, remoteMetacardsToUpdate);
-            throw new FederationAdminException(message, e);
+            throw new FederationAdminException("Error writing remote updates", e);
         }
     }
 
     private void createRemoteEntries(List<Metacard> remoteMetacardsToCreate)
             throws FederationAdminException {
         try {
-            Security.runAsAdminWithException(
-                    () -> federationAdminService.addRegistryEntries(remoteMetacardsToCreate, null));
+            Security.runAsAdminWithException(() -> federationAdminService.addRegistryEntries(
+                    remoteMetacardsToCreate,
+                    null));
+        } catch (PrivilegedActionException e) {
+            throw new FederationAdminException("Error creating remote entries", e);
```

Comment:
Does anybody care if any of these exceptions happen? Do we need to log something?